setup.py

```diff
@@ -31,8 +34,14 @@
     doc_requirements = [line.strip() for line in f]
 
 dev_requirements += doc_requirements
+full_requirements = extras_requirements
```

Comment:
So my understanding is full = base + extra, all = base + extra + dev. The naming is a bit confusing since "full" and "all" could both be seen as meaning everything.

Maybe "docs", "extended", and "full"? Others for "full" could be "beta" or "experimental".

Reply:
I just removed the "all". You can install both dev and extras using credoai_lens[full,dev]